pegasus/forms/hoc_signup_2017.rb

```diff
@@ -30,6 +30,23 @@ def self.normalize(data)
     result
   end
 
+  def self.process(data)
+    # If there is an nces school id then we need to load that school's address and use it as the event location.
+    # A school id of "-1" indicates that the school was not found in the school dropdown so we treat that the
+    # same as no school id.
+    nces_school_id = data['nces_school_s']
+    nces_school_name = ''
+    if nces_school_id && nces_school_id != "-1"
+      # id is the primary key of the schools table so we shouldn't get back more than one row
+      DASHBOARD_DB[:schools].where(id: nces_school_id).each do |school_data|
+        school_address = [school_data[:address_line1], school_data[:address_line2], school_data[:address_line3], school_data[:city], school_data[:state], school_data[:zip]].join(' ')
```

Comment:
Would this easier for reading the line?
```
[:address_line1, :address_line2, :address_line3, :city, :state, :zip].map {|k| school_data[k]}.join(' ')
```

Reply:
I reworked this with a variation of your suggestion that makes it easier for a reader to parse.